pages/collections/_type/index.vue

```diff
@@ -0,0 +1,132 @@
+          :total-results="total"
+          :per-page="perPage"
+          :link-gen="paginationLink"
+        />
+      </b-col>
+    </b-row>
+  </b-container>
+</template>
+
+<script>
+  import ContentHeader from '../../../components/generic/ContentHeader';
+  import ContentCard from '../../../components/generic/ContentCard';
+  import { pageFromQuery } from '../../../plugins/utils';
+  import {
+    getEntityIndex,
+    getEntitySlug,
+    getEntityTypeHumanReadable
+  } from '../../../plugins/europeana/entity';
+
+  const PER_PAGE = 24;
+  export default {
+    name: 'CollectionTypeIndexPage',
+    components: {
+      ContentHeader,
+      ContentCard,
+      PaginationNav: () => import('../../../components/generic/PaginationNav')
+    },
+    asyncData({ query, params, redirect, error, app }) {
+      const currentPage = pageFromQuery(query.page);
+      if (!['persons', 'topics'].includes(params.type)) {
+        return  error({ statusCode: 404, message: 'unknown collection type' });
+      }
+      if (currentPage === null) {
+        // Redirect non-positive integer values for `page` to `page=1`
+        query.page = '1';
+        return redirect(app.$path({ name: 'collections-type', params: { type: params.type }, query }));
+      }
+      const entityIndexParams = {
+        query: '*:*',
+        page: currentPage - 1,
+        type: params.type.slice(0, -1),
+        pageSize: PER_PAGE,
+        scope: 'europeana',
+        fl: 'skos_prefLabel.*,isShownBy,isShownBy.thumbnail'
+      };
+      return getEntityIndex(entityIndexParams)
+        .then(response => response)
+        .then(data => {
+          return {
+            entities: data.entities,
+            total: data.total,
+            page: currentPage,
+            perPage: PER_PAGE
+          };
+        })
+        .catch((e) => {
+          error({ statusCode: 500, message: e.toString() });
+        });
+    },
+    data() {
+      return {
+        perPage: PER_PAGE,
+        page: null
+      };
+    },
+    computed: {
+      showPagination() {
+        return this.total > this.perPage;
+      },
+      route() {
+        return {
+          name: 'collections-index',
+          params: {
+            pathMatch: this.$route.params.type
+          }
+        };
+      }
+    },
+    methods: {
+      paginationLink(val) {
+        return this.$path({ name: 'collections-type', params: { type: this.$route.params.type }, query: { page: val } });
+      },
+      entityRoute(entity) {
+        return {
+          name: 'collections-type-all',
+          params: {
+            type: getEntityTypeHumanReadable(entity.type),
+            pathMatch: getEntitySlug(entity.id, entity.prefLabel.en)
+          }
+        };
+      },
+      thumbnail(entity) {
+        if (typeof entity.isShownBy !== 'undefined') return entity.isShownBy.thumbnail;
+      }
+    },
+    head() {
+      return {
+        title: this.$tc(`entity.index.${this.$route.params.type}.title`, 2)
```

Comment:
The string from the lang file does not include pluralisation, so there is no need to use `$tc`. Use `$t` instead, without the numeric argument.

This also duplicates the code from the `ContentHeader` title attribute, so instead return it as `title` from `asyncData`, then use that in both places.